rfcs/0006-working-in-the-open.md

```diff
@@ -0,0 +1,104 @@
+---
+created-on: 2021-03-31
+freshness: 2021-03-31
+owner: @dj2
+status: proposal
+tags:
+  - opensource
+---
+
+[La version française suit.](#résumé)
+
+# Working in the open
+## Summary
+CDS as an organization has a commitment to working in the open, inspired by our
+peers in the UK, United States, and around the world. As we develop software
+products, coding in the open provides a number of important benefits:
+ * It encourages good practice
+ * It makes collaboration easier
+ * External users can help make it better
+ * Others can learn from our work
+ * It makes it easier to share standards
+ * It improves transparency on government’s work
+ * It clarifies ownership
+ * It helps make government technology seamless
+ * It’s easier to start code in the open than to open a closed repository
+
+This RFC attempts to lay out the _how_ of our open source work.
+
+
+## Motivation
+When we collaborate in the open and publish our data publicly, we can improve
+Government together. Building services more openly and publishing open data,
+simplifies access to government services and information, allows the public to
+contribute, and enables reuse by entrepreneurs, nonprofits, other agencies, and
+the public.
+
+CDS’s commitment to open source helps to normalize its use and creation across
+the Government of Canada. The [GC Digital Standards](https://www.canada.ca/en/government/publicservice/modernizing/government-canada-digital-standards.html)
+endorse the use and development of open source software within the Canadian
+Government.
+
+
+## Proposal
+* For new development choose a permissive license approved by the Open Source
+  Initiative, preferring the MIT License. MIT is the default license for USDS,
+  GDS, 18F, the Ontario Digital Service, and others.
```

Comment:
While I think pointing to other governments is a nice touch I think it would make more sense to also point to the GoCs guidance on choosing a license, or reference existing GoC Projects such as the web experience toolkit. 

If we are looking at these RFCs as something that OGD's can reference using existing GoC Guidance and examples will have more impact than trying to say that other jurisdictions are doing this thing. It makes it a lot harder to dismiss on the basis of their legislations, cultures, whatever are different then ours.

https://www.canada.ca/en/government/system/digital-government/digital-government-innovations/open-source-software/guide-for-publishing-open-source-code.html#toc04

https://github.com/wet-boew/wet-boew

Reply:
done